db/migrate/20220331120057_create_college_subjects.rb

```diff
@@ -0,0 +1,13 @@
+class CreateCollegeSubjects < ActiveRecord::Migration[6.1]
+  def change
+    create_table :college_subjects do |t|
+      t.string :name
+      t.text :description
+      t.references :departament
```

Comment:
Falta isso.

Para aplicar essa mudança, faz um pull e depois roda um `rails db:rollback` para desfazer a tabela e um migrate após para refazer. Isso vai atualizar o arquivo `db/schema.rb` também

```suggestion
      t.references :departament, foreign_key: true
```